wear/src/main/java/com/eveningoutpost/dexdrip/Models/BgReading.java

```diff
@@ -208,8 +208,8 @@ public synchronized static void updateDB() {
     }
 
     public double getDg_mgdl(){
-        if(dg_mgdl != 0) return dg_mgdl;
-        return calculated_value;
+        if(dg_mgdl != 0) return dg_mgdl;//from plugin
```

Comment:
These comments are not correct. Display glucose covers much more than just plugins, it handles noise compensation and more. I would suggest removing the comment at least from `dg_mgdl` as it is misleading. `calculated_value` is also not guaranteed to be the original xDrip algorithm either.

Reply:
ok, removed!